### PR TITLE
SimplE, SimplE_ignr support

### PR DESCRIPTION
**Pull request by menjarleev**
*Issue #, if available:*

*Description of changes:*

 [Results]
 * add SimplE, SimplE_ignr supports.
 * add optimal test results on 1-GPU, 8-GPU server in regards of FB15k, wn18 datasets
 * modify args to support SimplE, SimplE_ignr

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,16 +61,12 @@ Learn more details with our [documentation](https://aws-dglke.readthedocs.io/en/
...
-booktitle = {Proceedings of the 43rd International ACM SIGIR Conference on Research and Development in Information Retrieval},
-pages = {739–748},
-numpages = {10},
-series = {SIGIR '20}
+@misc{zheng2020dglke,
+    title={DGL-KE: Training Knowledge Graph Embeddings at Scale},
+    author={Da Zheng and Xiang Song and Chao Ma and Zeyuan Tan and Zihao Ye and Jin Dong and Hao Xiong and Zheng Zhang and George Karypis},
+    year={2020},
+    eprint={2004.08532},
+    archivePrefix={arXiv},
```
Comment (zheng-da) [inline README.md:69]: please don't change it back to the arxiv version.

**Comment by menjarleev** [comment]
Yeah, sorry I didn’t pull before push. I reckon I’ve fixed it already?

File: python/dglke/models/general_models.py [hunk 2]
```diff
@@ -27,9 +27,10 @@
 """
 import os
 import numpy as np
+import math
 import dgl.backend as F
 
-backend = os.environ.get('DGLBACKEND', 'pytorch')
+backend = os.environ.get('dglbackend', 'pytorch')
```
Comment (classicsong) [inline python/dglke/models/general_models.py:33]: This should be DGLBACKEND

**Review by classicsong — COMMENTED** [review]
Further, try to update the API docs: https://dglke.dgl.ai/doc/train.html
which should be in docs/source/train.rst.

**Comment by menjarleev** [comment]
sure. I’ll fix it up tmr

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: xiang song(charlie.song) <notifications@github.com>
发送时间: Monday, October 19, 2020 11:59:18 PM
收件人: awslabs/dgl-ke <dgl-ke@noreply.github.com>
抄送: Lingfei Huo <lhuo9710@uni.sydney.edu.au>; Author <author@noreply.github.com>
主题: Re: [awslabs/dgl-ke] SimplE, SimplE_ignr support (#157)


@classicsong commented on this pull request.

Further, try to update the API docs: https://dglke.dgl.ai/doc/train.html
which should be in docs/source/train.rst.

―

**Pull request closed without merging**